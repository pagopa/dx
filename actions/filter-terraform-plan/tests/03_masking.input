normal_value = "Keep Me"

hidden-link = "http://sensitive_test.link"
hidden-links = "http://non_sensitive_test.link"
very-hidden-link = "http://non_sensitive_test.link"

hiddden-link = "http://non_sensitive_test.link"

hidden-link: /app-insights-conn-string         = "InstrumentationKey=000000-000000-000000-000000;IngestionEndpoint=https://westeurope-3.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/;ApplicationId=000000-000000-000000-000000"
hidden-link: /app-insights-instrumentation-key = "000000-000000-000000-000000"
hidden-link: /app-insights-resource-id         = "/subscriptions/***/resourceGroups/XXXXX/providers/microsoft.insights/components/XXXXX"

test-sensible-key       = "test"
test-super-sensible-key = "test"
sensible-key            = "test"
hide-test-sensible-key  = "test"
test-sensible-key-hide  = "test"
