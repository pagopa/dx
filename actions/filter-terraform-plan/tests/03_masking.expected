Test mode enabled.
--- Executing Plan ---
normal_value = "Keep Me"

hidden-link = "[REDACTED]"
hidden-links = "http://non_sensitive_test.link"
